recipes/qnmfits/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "qnmfits" %}
+{% set version = "2024.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/qnmfits-{{ version }}.tar.gz
+  sha256: 13df4817c1dcf97b736938107bd652b292f92dc6f8a05a33c0e19b918a87398d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - scri >=2022.9.0
+    - qnm >=0.4.3
+
+test:
+  imports:
+    - qnmfits
+  requires:
+    - python {{ python_min }}
+
+about:
+  home: https://github.com/sxs-collaboration/qnmfits
```

Comment:
This URL is giving 404 errors. Is it public?

Reply:
The repository is now public and the link should work.

Reply:
Thanks!